client/starwhale/base/view.py

```diff
@@ -201,3 +191,124 @@ def _print_list(
                 )
 
         console.print(table)
+
+    @staticmethod
+    def pretty_json(data: t.Any) -> None:
+        console.print(Pretty(data, expand_all=True))
+
+    @staticmethod
+    def list_data(
+        _bundles: t.Dict[str, t.Any], show_removed: bool = False, fullname: bool = False
+    ) -> t.List[t.Dict[str, t.Any]]:
+        result = []
+        for _name, _versions in _bundles.items():
+            for _v in _versions:
+                if show_removed ^ _v["is_removed"]:
+                    continue
+
+                _version = (
+                    _v["version"]
+                    if fullname or show_removed
+                    else _v["version"][:SHORT_VERSION_CNT]
+                )
+                if _v.get("id"):
+                    _version = f"[{_v['id']:2}] {_version}"
+
+                result.append(
+                    {
+                        "name": _name,
+                        "version": _version,
+                        "tags": _v.get("tags", []),
+                        "size": _v.get("size", 0),
+                        "runtime": _v.get("runtime", ""),
```

Comment:
For the base view's list_data method, some cases may not include runtime column, such as `Runtime` object.